README.md

```diff
@@ -84,552 +84,284 @@ We conducted a comprehensive evaluation of Qwen2.5-Omni, which demonstrates stro
+| Refcocog+<sub>val</sub>  | **87.4**        | 87.2          | 86.1           | 75.2           | 
+| Refcocog+<sub>test</sub> | **87.9**        | 87.2          | 87.0           | 76.2           | 
+| ODinW                    | 42.4            | 37.3          | **55.0**       | 36.7           | 
+| PointGrounding           | 66.5            | **67.3**      | -              | -              | 
 
+</details>
 
 <details>
 <summary>Video(without audio) -> Text</summary>
 
 | Dataset                     | Qwen2.5-Omni-7B | Other Best | Qwen2.5-VL-7B | GPT-4o-mini | 
-|-----------------------------|--------------|------------|---------------|-------------|
-| Video-MME<sub>w/o sub</sub> | 64.3         | 63.9       | **65.1**      | 64.8        | 
-| Video-MME<sub>w sub</sub>   | **72.4**     | 67.9       | 71.6          | -           | 
-| MVBench                     | **70.3**     | 67.2       | 69.6          | -           | 
-| EgoSchema<sub>test</sub>    | **68.6**     | 63.2       | 65.0          | -           | 
+|-----------------------------|-----------------|------------|---------------|-------------|
+| Video-MME<sub>w/o sub</sub> | 64.3            | 63.9       | **65.1**      | 64.8        | 
+| Video-MME<sub>w sub</sub>   | **72.4**        | 67.9       | 71.6          | -           | 
+| MVBench                     | **70.3**        | 67.2       | 69.6          | -           | 
+| EgoSchema<sub>test</sub>    | **68.6**        | 63.2       | 65.0          | -           | 
+
 </details>
 
 <details>
 <summary>Zero-shot Speech Generation</summary>
 
+#### Content Consistency
+**Dataset:** SEED
+
+| Model              | test-zh | test-en | test-hard |
+|--------------------|---------|---------|-----------|
+| Seed-TTS_ICL       | 1.11    | 2.24    | 7.58      |
+| Seed-TTS_RL        | **1.00**| 1.94    | **6.42**  |
+| MaskGCT            | 2.27    | 2.62    | 10.27     |
+| E2_TTS             | 1.97    | 2.19    | -         |
+| F5-TTS             | 1.56    | **1.83**| 8.67      |
+| CosyVoice 2        | 1.45    | 2.57    | 6.83      |
+| CosyVoice 2-S      | 1.45    | 2.38    | 8.08      |
+| Qwen2.5-Omni-7B_ICL | 1.70    | 2.72    | 7.97      |
+| Qwen2.5-Omni-7B_RL  | 1.42    | 2.32    | 6.54      |
+
+#### Speaker Similarity
+**Dataset:** SEED
+
+| Model              | test-zh | test-en | test-hard |
+|--------------------|---------|---------|-----------|
+| Seed-TTS_ICL       | 0.796   | 0.762   | 0.776     |
+| Seed-TTS_RL        | **0.801**| **0.766**| **0.782**|
+| MaskGCT            | 0.774   | 0.714   | 0.748     |
+| E2_TTS             | 0.730   | 0.710   | -         |
+| F5-TTS             | 0.741   | 0.647   | 0.713     |
+| CosyVoice 2        | 0.748   | 0.652   | 0.724     |
+| CosyVoice 2-S      | 0.753   | 0.654   | 0.732     |
+| Qwen2.5-Omni-7B_ICL | 0.752   | 0.632   | 0.747     |
+| Qwen2.5-Omni-7B_RL  | 0.754   | 0.641   | 0.752     |
 
```

Comment:
Note that "Table 10: Single-Speaker Speech Generation" is missing from original markdown. Didn't add it, should I?
<img width="313" alt="Screenshot 2025-04-08 at 23 30 04" src="https://github.com/user-attachments/assets/b128661a-2d22-4fd6-bb7e-2a1262fbc69b" />